jsdoc/processors/check-access-tags.js

```diff
@@ -0,0 +1,44 @@
+module.exports = function checkAccessTagsProcessor(createDocMessage) {
+  return {
+    $runAfter: ['tags-extracted'],
+    $runBefore: ['processing-docs'],
+    $validate: { accessTypes: { presence: true } },
+    accessTypes: ['private', 'protected', 'public'],
+    $process: function(docs) {
+      var accessTypes = this.accessTypes;
+
+      docs.forEach(function(doc) {
+
+        // Check for invalid access type
+        if (doc.access && accessTypes.indexOf(doc.access) === -1) {
+          throw new Error(createDocMessage('"' + doc.access + '" is not an allowed access type for the "@access" tag. Try adding it to the checkAccessTagsProcessor.accessTypes array', doc));
+        }
+
+        // Check for multiple tags
+        var accessTypesFound = [];
+        accessTypes.forEach(function(accessType) {
+          if (doc[accessType]) {
```

Comment:
Yes, I am saying that you cannot have `@access private` and `@private` on the same doc.